### PR TITLE
[14.0][MIG] excel_import_export, excel_import_export_demo

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready for review.

### CODE REVIEW

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review. Works as expected on migrated env.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1: ,
Please you fix conflict file.

**Comment by kittiu** [comment]
I added support for multiple file export https://github.com/OCA/server-tools/pull/2022/commits/c9c6ea2379f949ef55e3cbe6311fad00f5dd6e21

I.e., from Sales Order, if user choose multiple file, it will zip them as "files.zip".

File: excel_import_export/models/xlsx_export.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+        outputs = []
+        for record in records:
+            self._fill_workbook_data(wb, record, export_dict)
+            # Return file as .xlsx
+            content = BytesIO()
+            wb.save(content)
+            content.seek(0)  # Set index to 0, and start reading
+            out_file = content.read()
+            if record and "name" in record and record.name:
+                out_name = record.name.replace(" ", "").replace("/", "")
```
Comment (Saran440) [inline excel_import_export/models/xlsx_export.py:261]:
```suggestion
                out_name = record.name.replace(" ", "").replace("/", "")
```

Please, Could you add function get name this line for manual name export file excel?

File: excel_import_export_demo/import_export_sale_order/templates.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<!--
+  Copyright 2019 Ecosoft Co., Ltd.
+  License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="sale_order_xlsx_template" model="xlsx.template">
+        <field name="res_model">sale.order</field>
+        <field name="fname">sale_order.xlsx</field>
+        <field name="name">Sale Order Template (import/export)</field>
+        <field name="description">Sample Sales Order Template for testing</field>
+        <field name="import_action_id" eval="ref('action_sale_oder_export_xlsx')" />
```
Comment (Saran440) [inline excel_import_export_demo/import_export_sale_order/templates.xml:11]:
```suggestion
        <field name="import_action_id" eval="ref('action_sale_oder_import_xlsx')" />
        <field name="export_action_id" eval="ref('action_sale_oder_export_xlsx')" />
```

**Review by Saran440 — COMMENTED** [review]
Reviewed.

**Comment by dreispt** [comment]
Can you fix red build please?

**Comment by kittiu** [comment]
Waiting for fix -- https://github.com/OCA/server-tools/pull/2094

**Comment by kittiu** [comment]
@dreispt please help merge again. Thank you!

**Review by newtratip — APPROVED** [review]

**Comment by kittiu** [comment]
@dreispt please, can you try to merge again. I have fixed the same thing as to make v13 PR mergeable. https://github.com/OCA/server-tools/compare/2c17b07044f0b72bc830772e974b0c4f42377806..32877ec3f6fee5ec30c717189a2e0f609c3ede23
Should be fine this time. Thank you :pray:

**Pull request merged**